pkg/kubefed2/unjoin.go

```diff
@@ -285,14 +285,12 @@ func deleteFederatedClusterAndSecret(hostClientset kubeclient.Interface, client
 func deleteRBACResources(unjoiningClusterClientset kubeclient.Interface,
 	namespace, unjoiningClusterName, hostClusterName string, dryRun bool) bool {
 
-	var deletionSucceeded = true
```

Comment:
What is golint complaining about in this instance?

Reply:
```
ineffectual assignment to `deletionSucceeded`
```

the statement to initialzie `deletionSucceeded` var is not useful, which would be overwritten after calling `deleteClusterRoleAndBinding`

Reply:
I would prefer that this check be disabled.  An 'ineffectual assignment' may well be useful for communicating intent to future maintainers even if it has no value to the golang runtime.